src/Discord.Net.Rest/API/Rest/ModifyGuildWelcomeScreenParams.cs

```diff
@@ -0,0 +1,16 @@
+using Newtonsoft.Json;
+
+namespace Discord.API.Rest;
+
+[JsonObject(MemberSerialization = MemberSerialization.OptIn)]
```

Comment:
```suggestion
```

Reply:
This attribute has no effect on the final json produced when serializing this class